File: Exercise/Medium/04. Count Words in a Sentence/Code/Count Words in a Sentence.cs
using System;

class Program {
    static int CountWords(string line) {
        string[] words = line.Split(new char[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
        return words.Length;
    }

    static void Main() {
        string line = Console.ReadLine();
        Console.WriteLine(CountWords(line));
    }
}

File: Exercise/Medium/04. Count Words in a Sentence/Code/Count Words in a Sentence.java
import java.util.Scanner;

public class Main {
    static int countWords(String line) {
        String[] words = line.split("\\s+");
        return words.length;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String line = scanner.nextLine();
        scanner.close();

        System.out.println(countWords(line));
    }
}



